packages/xo-web/src/common/icon.js

```diff
@@ -23,3 +23,21 @@ Icon.propTypes = {
 }
 
 export { Icon as default }
+
+export const StackedIcons = ({ icons, ...props }) => (
+  <span className={`fa-stack ${props.className}`}>
+    {icons.map(({ icon, size, ...iconProps }) => (
+      <Icon
+        {...iconProps}
+        {...props}
```

Comment:
You can move it to the wrapper component to be able to customize it and to customize each icon. It will add more flexibility.
```js
StackedIcons = ({ icons, className ...props }) => (
  <span className={`fa-stack ${className}`} {...props}>
```